CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# v1.0.3 (2020-11-19)
+
+## OS Changes
+* Support setting Linux kernel parameters (sysctl) via settings ([#1158], [#1171])
+* Add `bpffs` mount to default installation ([#1160], **thanks @izaakschroeder!**)
+* Create links for ephemeral storage devices ([#1173])
+* Package updates and patches ([#1176], [#1195], [#TBD])
+* Set default RLIMIT_NOFILE in CRI  ([#1180])
```

Comment:
Let's indicate that the new default is a 65536 soft limit on open file descriptors, and a 1048576 hard limit.